lib/Command/Clean.php

```diff
@@ -0,0 +1,221 @@
+
+	private ?Row $row = null;
+	private int $offset = -1;
+
+	public function __construct(LoggerInterface $logger, ColumnService $columnService, RowService $rowService, TableService $tableService, RowMapper $rowMapper) {
+		parent::__construct();
+		$this->logger = $logger;
+		$this->columnService = $columnService;
+		$this->rowService = $rowService;
+		$this->tableService = $tableService;
+		$this->rowMapper = $rowMapper;
+	}
+
+	protected function configure(): void {
+		$this
+			->setName('tables:clean')
+			->setDescription('Clean the tables data.')
+			->addOption(
+				'dry-run',
+				'd',
+				InputOption::VALUE_NONE,
+				'Prints all wanted changes, but do not write anything to the database.'
+			)
+			->addOption(
+				'only-errors',
+				'e',
+				InputOption::VALUE_NONE,
+				'Show only errors.'
+			)
+		;
+	}
+
+	/**
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$this->dry = !!$input->getOption('dry-run');
+		$this->showOnlyErrors = !!$input->getOption('only-errors');
+
+		if ($this->dry) {
+			$this->print("Dry run activated.");
+		}
+		if ($this->showOnlyErrors) {
+			$this->print("Print only errors mode activated.");
+		}
+
+		// check action, starting point for magic
+		$this->checkIfColumnsForRowsExists();
+
+		return 0;
+	}
+
+	private function getNextRow():void {
+		try {
+			$this->row = $this->rowMapper->findNext($this->offset);
+			$this->offset = $this->row->getId();
+		} catch (MultipleObjectsReturnedException|Exception $e) {
+			$this->print('Error while fetching row', self::PRINT_LEVEL_ERROR);
+		} catch (DoesNotExistException $e) {
+			$this->print("");
+			$this->print("No more rows found.", self::PRINT_LEVEL_INFO);
+			$this->print("");
+			$this->row = null;
+		}
+	}
+
+
+	/**
+	 * Take each data set from all rows and check if the column (mapped by id) exists
+	 *
+	 * @return void
+	 */
+	private function checkIfColumnsForRowsExists(): void {
+
+		$this->getNextRow();
+		while ($this->row) {
+			$this->print("");
+			$this->print("");
+			$this->print("Lets check row with id = " . $this->row->getId());
+			$this->print("==========================================");
+
+			$this->checkColumns();
+
+			$this->getNextRow();
+		}
+	}
+
+	private function checkColumns(): void {
+		$data = json_decode($this->row->getData());
+		foreach ($data as $date) {
+			$suffix = strlen($date->value) > $this->truncateLength ? "...": "";
+			$this->print("");
+			$this->print("columnId: " . $date->columnId . " -> " . substr($date->value, 0, $this->truncateLength) . $suffix, self::PRINT_LEVEL_INFO);
+
+			try {
+				$this->columnService->find($date->columnId, '');
+				if(!$this->showOnlyErrors) {
+					$this->print("column found", self::PRINT_LEVEL_SUCCESS);
+				}
+			} catch (InternalError $e) {
+				$this->print("😱️ internal error while looking for column", self::PRINT_LEVEL_ERROR);
```

Comment:
Are those logged elsewhere, otherwise it might be nice to still put the exception into the log for further details.